ofono/include/lte.h

```diff
@@ -38,7 +38,7 @@ typedef void (*ofono_lte_cb_t)(const struct ofono_error *error, void *data);
 
 struct ofono_lte_driver {
 	const char *name;
-	int (*probe)(struct ofono_lte *lte, void *data);
+	int (*probe)(struct ofono_lte *lte, unsigned int vendor, void *data);
```

Comment:
I don't like that since it breaks plugin API. Upstream plugin API is considered unstable - we can't afford that.
On the other hand I'm not aware of any external plugin actually using this API...
IIRC this was one of the reasons why I stopped at 1.23

Reply:
Ah okay. I wasn't aware of that to be honest. I'm completely new to Sailfish actually. My only experience with Ofono before is in the use of Plasma Mobile on the PinePhone. Over there we use version 1.31, which for the PinePhone, which at the moment is quite stable. Of course the PinePhone isn't really comparable with most other devices of course, and Plasma Mobile also doesn't use the SFOS specific stuff.